content/blog/trungnt2910/2024-07-28_[gsoc_2024]_debugging_progress_2.md

```diff
@@ -2,7 +2,7 @@
 type = "blog"
```

Comment:
Renaming this file will change the permalink and thus break the forum thread, I think? Probably just leave the file name as-is, the date embedded in it is correct at least.

Reply:
Isn't it better to fix it once and for all then make it permanently wrong? It's not a critical and popular post that needs to be rock stable... There aren't info that is too important on the thread as well.

Reply:
I think we should just err on the side of preserving the comments. By the same argument as it not being critical, it's not critical that it have the right URL either I think.

Reply:
Wouldn't the "new" post generate its own forum thread, where admins can transfer comments to?

Such a transfer would confuse people once, a wrong link would confuse people (and potentially tools) every time.

Better fix this as soon as possible.

Reply:
OK, I suppose so.